File: AdrenalinClient/src/components/Clubs/Club.js
import React from 'react';
import styles from './clubs.module.css';

export default function Club({ clubItem }) {
  return (
    <div className={styles.clubItemWrapper}>
      <img
        src={require('../../assets/img/' + clubItem.image)}
        alt="new-image"
      ></img>
      <div className={styles.textBlock}>
        <p>{clubItem.city}</p>
        <p>{clubItem.adress}</p>
      </div>
    </div>
  );
}

File: AdrenalinBackend/Domain/Models/Club.cs
using Domain.Models.Base;

namespace Domain.Models
{
    public class Club : BaseEntity
    {
        public string? Image { get; set; }
        public string? City { get; set; }
        public string? Adress { get; set; }

        public List<Visit> Visits { get; set; }
    }
}
